tests/testthat/test-config-log.R

```diff
@@ -73,7 +73,7 @@ teardown({ cleanup() })
 withr::with_options(list(rbabylon.model_directory = NULL), {
 
   test_that("config_log() errors with no .base_dir set", {
-    expect_error(config_log(), regexp = "`.base_dir` cannot be `NULL`")
+    expect_error(config_log(), regexp = "argument.+is missing")
```

Comment:
I think we can just delete this test. We were checking a code path, but now we're just checking... base R's behavior when you forget to pass a required argument? Seems unnecessary.